requirements.txt

```diff
@@ -8,7 +8,8 @@ Jinja2>=2.7.3
 Sphinx>=1.2.3
 docopt>=0.6.2
 simplejson>=3.8.1
+requests>=2.9.1
 
 # Python3 compat
 future>=0.15.2
-configparser>=3.3.0; python_version < '3'
+#configparser>=3.3.0; python_version < '3'
```

Comment:
BTW for me the tool is working *without* having a python-configparser tool !! So for which feature do we need it ?